starterOnly/style.css

```diff
@@ -0,0 +1,32 @@
+.error {
+    margin: 6px 0;
+    font-size: 14px;
+    color: red;
+    font-weight: thin;
+  }
+  .checkbox2-label {
+    margin-top: 11px;
+  }
+  div.content {
+    min-height: 625px;
+  }
+  #close-btn {
```

Comment:
try to use classname and not ids'

Reply:
Changing to classes make errors at the moment